cmd/cmd-run.go

```diff
@@ -209,7 +213,20 @@ func run(cmd *cobra.Command, _ []string) error {
 
 		CreateGit: gitCreator,
 	}
-
+	if repoInclude != "" {
```

Comment:
This was now moved to be further from the rest of the flag parsing. Please keep it together with the rest. Move it here https://github.com/lindell/multi-gitter/blob/b41d462b1362e48a6438b36818dc9f8a322cf81f/cmd/cmd-run.go#L151